packages/protocol/contracts/tokenvault/BridgedERC20.sol

```diff
@@ -1,39 +1,56 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.24;
 
-import "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC20VotesUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/introspection/IERC165Upgradeable.sol";
+import "../common/EssentialContract.sol";
+import "../common/LibStrings.sol";
+import "./IBridgedERC20.sol";
 import "./LibBridgedToken.sol";
-import "./BridgedERC20Base.sol";
-
-/// @notice BridgedERC20 was `BridgedERC20Base, ERC20SnapshotUpgradeable, ERC20VotesUpgradeable`.
-/// We use this contract to take 50 more slots to remove `ERC20SnapshotUpgradeable` from the parent
-/// contract list.
-/// We can simplify the code since we no longer need to maintain upgradability with Hekla.
-// solhint-disable contract-name-camelcase
-abstract contract BridgedERC20Base_ is BridgedERC20Base {
-    // solhint-disable var-name-mixedcase
-    uint256[50] private __slots_previously_used_by_ERC20SnapshotUpgradeable;
-}
 
 /// @title BridgedERC20
 /// @notice An upgradeable ERC20 contract that represents tokens bridged from
 /// another chain.
+/// Note this contract offers timestamp-based checkpoints and voting functions.
 /// @custom:security-contact security@taiko.xyz
-contract BridgedERC20 is BridgedERC20Base_, ERC20VotesUpgradeable {
+contract BridgedERC20 is EssentialContract, IBridgedERC20, ERC20Upgradeable, IERC165Upgradeable {
```

Comment:
Ok, so the idea is not to have Votes anymore in Bridged tokens ? If so the note above:
`Note this contract offers timestamp-based checkpoints and voting functions.` -> Seems to be not correct as we dont have voting.
Also how do we ensure the upgrade from bridged tokens (alread on hekla) ? Or we simply dont upgrade the old tokens, but from X point on (X is where the impl. is deployed on Hekla) new tokens bridged from Hoelsky -> Hekla will use this deployed implementation behind the proxy ?

Reply:
The comment removed.
Correct, for deployed bridged tokens on Hekla, we leave them as-is.